src/app/catalog/catalog.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams, HttpHeaders } from '@angular/common/http';
+
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+import { Book } from './catalog';
+import { LocalstorageService } from '../services/localstorage.service';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class CatalogService {
+  catalogUrl = 'https://js-band-store-api.glitch.me/books';
+
+  constructor(
+    private http: HttpClient,
+    private localStorageService: LocalstorageService,
+    private router: Router
+  ) {}
+
+  getCatalog(): Observable<Book[]> {
```

Comment:
Naming should be meaningfull, you are fetching books, so the method should called getBooks()